.circleci/config.yml

```diff
@@ -295,6 +295,42 @@ commands: # reusable commands with parameters
       - *store-log-artifacts
       - *upload-coverage
 
+  install-openshift:
+    steps:
+      - run:
+          name: Install OpenShift Client Tools
+          working_directory: /tmp
+          command: |
+            curl --fail -L  https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz | tar -xzf-
```

Comment:
Why 3.11? Better OpenShift 4.X, no?

Reply:
Haven't tried that yet. Not even for local dev. 

You need to use the codeready containers thing.
https://developers.redhat.com/blog/2019/09/05/red-hat-openshift-4-on-your-laptop-introducing-red-hat-codeready-containers/

You need lots of resources.

* 4 virtual CPUs (vCPUs)
* 8 GB of memory
* 35 GB of storage space